cts/cli/regression.daemons.exp

```diff
@@ -1,435 +1,747 @@
 =#=#=#= Begin test: Get CIB manager metadata =#=#=#=
-<?xml version=""?>
```

Comment:
This is required for XML 1.1 documents (https://www.w3.org/TR/xml11/#sec-prolog-dtd) but apparently not for older versions.  I don't know which we are using offhand, but I'd lean towards making sure it's included just to be clear.

Reply:
Our API output already doesn't include it though.
```
reid@laptop:~/git/pacemaker$ sudo crm_mon --output-as=xml
<pacemaker-result api-version="2.32" request="crm_mon --output-as=xml">
  <summary>
...
```

The goal is to be using an XML `pcmk__output_t` object by the time this PR is finished; this is just an intermediate step.

Reply:
The output object should probably display it, but that doesn't need to be part of this

Reply:
I'm fine with that, but yeah if we display it then it needs to be done at the output object level.

Reply:
https://projects.clusterlabs.org/T767

Reply:
> This is required for XML 1.1 documents (https://www.w3.org/TR/xml11/#sec-prolog-dtd) but apparently not for older versions. I don't know which we are using offhand, but I'd lean towards making sure it's included just to be clear.

For an unrelated reason, I just got confirmation that libxml only supports XML 1.0.
https://discourse.gnome.org/t/does-xmlnewdoc-support-xml-1-1/19329